### PR TITLE
feat: add "plain" variant to Accordion

### PR DESCRIPTION
**Pull request by tujoworker**
This PR adds a new variant to the Accordion component.

- For now, we simply drop any styles.
- Add types support for it.
- Add both visual and integration test.
- The example is (for now) not visible in the docs, as it is an edge case. But it is used for the screenshot test.

Screenshot 2022-04-26 at 13 07 10

DNB Eienedom has requested this.

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/components/accordion/properties.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ These properties can send along with the `Accordion.Provider` or `Accordion.Grou
 | `remember_state`                            | _(optional)_ if set to `true`, it will remember a changed state initiated by the user. It requires a unique `id`. It will store the sate in the local storage.                                                                               |
 | `flush_remembered_state`                    | _(optional)_ if set to `true`, the saved (remembered) will be removed and the initial component state will be used and set.                                                                                                                  |
 | `no_animation`                              | _(optional)_ if set to `true`, the open and close animation will be omitted.                                                                                                                                                                 |
+| `variant`                                   | _(optional)_ defines the used styling. As of now, only `outlined` is available. Use `palin` for no styles. It defaults to `outlined`.                                                                                                        |
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/components/accordion/properties.md:18]:
```suggestion
| `variant`                                   | _(optional)_ defines the used styling. As of now, only `outlined` is available. Use `plain` for no styles. It defaults to `outlined`.                                                                                                        |
```

**Review by langz — APPROVED** [review]
Great having a screenshot test for this :)
